project/SbtUpdatesSettings.scala

```diff
@@ -6,7 +6,7 @@ import sbt.Keys.*
 object SbtUpdatesSettings {
 
   lazy val sbtUpdatesSettings = Seq(
-    dependencyUpdatesFailBuild := true,
+    dependencyUpdatesFailBuild := false,
```

Comment:
this should be `true` right?

Reply:
good catch, yeah it should be

Reply:
I actually did not change that, so I do not know why that came through. must have been something leftover from me testing something before this? changing back now